instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpCommonAttributesGetter.java

```diff
@@ -47,6 +107,27 @@
    *
    * <p>Implementations of this method <b>must not</b> return a null value; an empty list should be
    * returned instead.
+   *
+   * @deprecated This method is deprecated and will be removed in a future release. Implement {@link
+   *     #getHttpResponseHeader(Object, Object, String)} instead.
+   */
+  @Deprecated
+  default List<String> getResponseHeader(REQUEST request, RESPONSE response, String name) {
+    return emptyList();
+  }
+
+  // TODO: make this required to implement
+  /**
+   * Returns all values of header named {@code name} from the response, or an empty list if there
+   * were none.
+   *
+   * <p>This is called from {@link Instrumenter#end(Context, Object, Object, Throwable)}, only when
+   * {@code response} is non-{@code null}.
+   *
+   * <p>Implementations of this method <b>must not</b> return a null value; an empty list should be
+   * returned instead.
    */
-  List<String> getResponseHeader(REQUEST request, RESPONSE response, String name);
+  default List<String> getHttpResponseHeader(REQUEST request, RESPONSE response, String name) {
```

Comment:
I'm unclear on the goal behind renaming all the methods to have `http` in the name. The class itself is clearly http  related, so I think it feels a little redundant.

Reply:
Yup -- it's a conscious decision that we made, documented [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8491#issuecomment-1564579851). [Here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8491#discussion_r1196040376)'s some context about that (TL;DR the HTTP getters will extend other getters, and we want the method names to reflect the attributes they represent)

Reply:
Cool, thanks for clarifying. Was also mentioned in the SIG call today. 👍🏻